hack/app_sre_pr_check.sh

```diff
@@ -4,4 +4,10 @@ set -ex
 
 cd $(dirname $0)/..
 
+#convert yaml to json
+find . -name 'metadata.yaml' -exec python3 hack/yamltojson.py scripts/SREP/example/metadata.yaml  {} \;
```

Comment:
why you put "scripts/SREP/example/metadata.yaml" here?

Reply:
I am passing it as argument

Reply:
You need to remove that `scripts/SREP/example/metadata.yaml`

Reply:
removed